packages/wrangler/src/__tests__/api-dev.test.ts

```diff
@@ -16,4 +20,182 @@ describe("unstable_dev", () => {
 		}
 		await worker.stop();
 	});
+
+	describe("protocol", () => {
+		it("should use parse to give https url with localProtocol", async () => {
+			const [input, _] = parseRequestInput(
```

Comment:
Can we split out the tests for helper functions like `parseRequestInput` into their own test file?

Just wanted to keep `api-dev.test.ts` for testing `unstable_dev` directly

Reply:
https://github.com/cloudflare/wrangler2/pull/1839/commits/63a3fe11f767e9c27fe3dd6bdd2da517d7302ae7